PyTado/http.py

```diff
@@ -185,13 +182,11 @@ def __init__(
         self._expires_at: datetime | None = None
 
         self._id: int | None = None
-        self._token_refresh: str | None = None
         self._x_api: bool | None = None
-        self._token_file_path = token_file_path
 
-        if saved_refresh_token or self._load_token():
-            if self._refresh_token(refresh_token=saved_refresh_token, force_refresh=True):
-                self._device_ready()
+        saved_refresh_token = self._token_manager.load_token()
+        if saved_refresh_token and self._refresh_token(refresh_token=saved_refresh_token, force_refresh=True):
+            self._device_ready()
```

Comment:
@erwindouna this lines where wrong, in case the _refresh_token method failed, we wouldn't go into _login_device_flow. I fixed that now!